lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,55 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
+			which should be assumed to be already loaded (or 'provided') by the environment into which bundle module is loaded.
+ * 	</li>
+ *	<li>
+		 <code>raw</code>: A raw section determines the set of modules that should be embedded,
+			sorts them according to their dependencies and writes them out 1:1 without any transformation or wrapping (raw). Only JavaScript sources
+				can be embedded in a raw section.
+	</li>
+ *	<li>
+		 <code>preload</code>: A preload section packages resources that should be stored in the preload cache in the client.
+			They can embed any textual resource type (JavaScript, CSS and Theme-Parameter files) that the bundling supports. The modules are wrapped into a call to jQuery.sap.registerPreload()
+				and each Javascript module will be embedded in an anonymous function.
+	</li>
+ *	<li>
+		 <code>require</code>: A require section is transformed into a sequence of jQuery.sap.require calls. It only has text content, no nested options.
+			The text content must contain a comma or whitespace separated list of module names or patterns. The list will be resolved like an include pattern list
+				in any of the other sections and for each of the resolved modules, a jQuery.sap.require will be created.
+	</li>
+ * </ul>
+ * </p>
+ *
+ * @public
+ * @typedef {object} ModuleBundleDefinitionSection
+ * @property {string} mode The embedding technology. Either 'provided', 'raw', 'preload' or 'require'
+ * @property {string[]} filters List of modules declared as glob patterns that should be in- or exluded.
+ * 		Exludes should be marked with a leading exclamation mark.
+ * @example <caption>List of modules as glob patterns that should be in- or excluded</caption>
+ * filters:
+ * - "some/path/to/module"
+ * - "some/other/path/to/module"
+ * - "!some/path/to/module/to/be/excluded"
+ * - "!some/other/path/to/module/to/be/excluded"
+ * @property {boolean} [resolve=false] Whether (transitive) dependencies of module should be resolved
```

Comment:
of module --> of modules that match the given filters

Reply:
Done